crates/graphql/src/schema/objects/graph_connection.rs

```diff
@@ -33,6 +33,8 @@ impl From<models::TwitterEnrichedGraphConnection> for GraphConnection {
             to_account,
             from_twitter_handle,
             to_twitter_handle,
+            connected_at: _,
+            disconnected_at: _,
```

Comment:
Do we want to expose these fields in GraphQL? cc: @kespinola

Reply:
connected_at be a good one. don't need disconnect. we'll use that to filter out results.

Reply:
Should be exposed now